underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystemFactory.java

```diff
@@ -68,34 +68,34 @@ public boolean supportsPath(String path) {
    *
    * @return true if simulation mode or if all required authentication credentials are present
    */
-  private boolean addAndCheckSwiftCredentials() {
+  private boolean addAndCheckSwiftCredentials(UnderFileSystemConfiguration conf) {
```

Comment:
I think the better way is to put it back to ufs conf.
However, if we make this change, I am not sure if I will break any assumption in Swift UFS.
@madanadit , can you double check? (i.e., if I add it back to ufs conf, rather than configuration, will I break anything)?